pyproject.toml

```diff
@@ -143,7 +143,7 @@ exclude_also = [
 ]
 
 [tool.typos.default]
-extend-words = { adaptee = "adaptee" }
+extend-words = { adaptee = "adaptee", "arange" = "arange" }
```

Comment:
I would drop the quotes for the key for consistency (`arange = "arange"`) or use this variant for extra inline comment:

```toml
[tool.typos.default]
extend-ignore-identifiers-re = ["NDArray*", "interm", "af000ded"]

[tool.typos.default.extend-words]
adaptee = "adaptee"
# Commonly used via `from torch import arange`
arange = "arange"
```

Reply:
good idea, let's do that